app.js

```diff
@@ -1,30 +1,30 @@
 var createError = require('http-errors');
 var express = require('express');
 var session = require('express-session'); 
-var MySQLStore = require('express-mysql-session')(session);   // import and add mysql session by creating a new
+var MySQLStore = require('express-mysql-session')(session);
 var app = module.exports = express();
 var path = require('path');
 var cookieParser = require('cookie-parser');
 var logger = require('morgan');
+require('dotenv').config();
 
 var indexRouter = require('./routes/index');
 var usersRouter = require('./routes/users');
 
 var app = express();
 
-var options = {
-  host: '${process.env.SESSION_HOST}',
-  port: '${process.env.SESSION_PORT}',
-  user: '${process.env.SESSION_USER}',
-  password: '${process.env.SESSION_PASSWORD}',
-  database: '${process.env.SESSION_DATABASE}'
-};
-// adfawefaer
-var sessionStore = new MySQLStore(options);
+// getter function for environment variables
+function getOption(varname){
+  return process.env[varname]; 
+}
+
+console.log(getOption("SESSION_HOST"));
+
+var sessionStore = new MySQLStore(getOption);
```

Comment:
Looks like here you're passing `MySQLStore` the function `getOption` rather than the expected object. I'm not sure if MySQLStore will know what to do with the passed function when it is expecting `options` to be an object containing [these values](https://www.npmjs.com/package/express-mysql-session#options)